app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteListFragment.kt

```diff
@@ -133,8 +133,10 @@ open class AutocompleteListFragment : Fragment() {
         val removeItem = menu?.findItem(R.id.remove)
 
         removeItem?.isVisible = isSelectionMode() || domainList.adapter.itemCount > 1
-        removeItem?.isEnabled = !isSelectionMode()
+        val isEnabled = !isSelectionMode()
                 || (domainList.adapter as DomainListAdapter).selection().isNotEmpty()
+        removeItem?.isEnabled = isEnabled
+        removeItem?.icon?.alpha = if (isEnabled) SettingsFragment.ALPHA_ENABLED else SettingsFragment.ALPHA_DISABLED
```

Comment:
nit: If it works for you, works for me so r+ but can we use [`android:tintList`](https://developer.android.com/reference/android/view/MenuItem.html#attr_android:iconTint) instead?

Reply:
After your suggestion I gave that a try (copied the state list into @color, used app:itemIconTint) and while that works for r25...it doesn't for r23.

I also tried setting the tint programmatically but at that point, since we're invalidating every time, it's not really worth it.